DiscordRPC/Configuration.cs

```diff
@@ -24,9 +24,9 @@ public class Configuration
 		public string CdnHost { get; set; }
 
 		/// <summary>
-		/// The type of enviroment the connection on. Usually Production. 
+		/// The type of environment the connection on. Usually Production. 
 		/// </summary>
-		[JsonProperty("enviroment")]
-		public string Enviroment { get; set; }
+		[JsonProperty("environment")]
```

Comment:
So this is confirmed that it is `environment` in the API?

Reply:
Yup.
![image](https://user-images.githubusercontent.com/19228318/204102333-528f75db-d40e-4225-9e94-cd3ea63aa9e5.png)